src/redux/categories/categories.js

```diff
@@ -0,0 +1,12 @@
+const CHECK_STATUS = 'CHECK_STATUS';
+
+export default function categoriesReducer(state = { categories: [] }, action = {}) {
+  switch (action.type) {
+    case CHECK_STATUS: return state.categories.push('Under construction');
+    default: return state;
+  }
+}
+
+export function checkStatus(status) {
+  return { type: CHECK_STATUS, status };
+}
```

Comment:
- [Optional] What do you think of creating action types first before reducers?. Most documentation follows this methodology.